### DISCUSSION TITLE
ADD NEW DEVICES UT60BT

### DISCUSSION THREAD

**Discussion by shivram1974** [webspiderteam/Bluetooth-DMM-For-Windows#55] [General]
HAI SIR PLEASE ADD UT60BT FROM UNI-T ALSO BLUETOOTH DMM NEED PC CONNECT HELP ME THANKS FOR YOUR GOOD PROJECT

**Comment by webspiderteam** [comment]
Sure  can add if it supports ble. But  need your help because  need sample data to test it and sone device info that we can gatter from your device. if you can help me about these  can add it in a few days. First of all  need uuids.txt in app folder to find gatt uuids. can you send me? To create uuids.txt you only need to connect your device with my app. it doesnt matter which device type you chose. You will not see any readings but it will give you this file. Also there will be a file named as lie "log-2024-03-28.txt" if its generated it will give us sample data too.

**Comment by ludwich66** [comment]
Did you know this Site?
https://github.com/ljakob/unit_ut61eplus
http://sigrok.org/wiki/UNI-T_UT61B
https://lukasschwarz.de/ut61b
https://github.com/cromarty/dmm

Grüße ludwich

**Reply by webspiderteam** [reply]
Usb protocolcs or rs232 protocols can be different but is there is a chance to be similir thar would help me so much. As always you did alot again. Thank you so much @ludwich66.

**Reply by shivram1974** [reply]
UT60BT USING BT V.4.00 ALSO IN PC APP FOLDER ICANT FIND uuids.txt FILE ALSO MOBILE APP TOO

**Reply by shivram1974** [reply]
SHARE YOUR WHAT'S APP NO I CAN CHAT WITH WHAT FILE YOU WANT MY NO:+919443019097

**Reply by webspiderteam** [reply]
![image](https://github.com/webspiderteam/Bluetooth-DMM-For-Windows/assets/85828505/23ec7992-1223-42d2-9280-804fe235dafe)
these files especially uuids.txt. app shoud create it when you try to connect your device. This file will give us gatt Uuid for your device type then we can gett data from device.

**Comment by rappbledor** [comment]
I am trying to use a UNI-T 60BT DMM. It appears in the list, and I'm able to "Connect" to it, it will even offer to Auto-Re-Connect if I close the application, but when I try to take a reading nothing happens and it stays at zero despite numbers on the display of the meter.

 There's a MODE section, but none of the buttons work, and cause a crash error.
[crashlog_19-07-2025_00_22_00.txt](https://github.com/user-attachments/files/21324699/crashlog_19-07-2025_00_22_00.txt)
[log-2025-07-19.txt](https://github.com/user-attachments/files/21324700/log-2025-07-19.txt)
[uuids.txt](https://github.com/user-attachments/files/21324701/uuids.txt)

Let me know what else I can do/what other info you need.    I don't expect the MODE switch will ever work on mine, but I do want the reporting.

**Reply by webspiderteam** [reply]
Thank you for sharing logs. I am going to look at it and probably i can fix. I didnt have any log for this device now i have al least 1 line :) I wil try to fix it as soon as posible. and send you fixed version from this topic.

**Reply by webspiderteam** [reply]
Sorry i forgot that we already wored on it with shivram and it was woring perfect. maybe we need to checc with debug app 
 
[Debug.zip](https://github.com/user-attachments/files/21327144/Debug.zip)
try with this one this also creates same log files. if it wors or not they will help me to understand situation. send them to me pls.
